database/src/main/kotlin/no/nav/su/se/bakover/database/søknad/SøknadPostgresRepo.kt

```diff
@@ -27,4 +29,24 @@ internal class SøknadPostgresRepo(
         }
         return hentSøknad(søknad.id)!!
     }
+
+    override fun lukkSøknad(søknadId: UUID, lukket: Søknad.Lukket) {
+        dataSource.withSession { session ->
+            "update søknad set lukket=to_json(:lukket::json) where id=:id".oppdatering(
+                mapOf(
+                    "id" to søknadId,
+                    "lukket" to objectMapper.writeValueAsString(lukket.toJson())
+                ),
+                session
+            )
+        }
+    }
+
+    override fun harSøknadPåbegyntBehandling(søknadId: UUID): Boolean {
```

Comment:
Grunnen til at vi flyttet den hit var for å unngå å la a) søknad bruke behandlingsRepo direkte og b) ikke la søknadService/behandlingService få en syklisk referanse til hverandre